GREEDY/DataManagers/ImageFormatting.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AForge;
+using AForge.Imaging.Filters;
+using Emgu.CV;
+using Emgu.CV.CvEnum;
+using Emgu.CV.Structure;
+using Emgu.CV.Util;
+using Point = System.Drawing.Point;
+
+namespace GREEDY.DataManagers
+{
```

Comment:
This code is really complex. Maybe it could be broken down into smaller pieces with better explanations? Good work anyways ;)